skore/src/skore/cli/cli.py

```diff
@@ -66,21 +68,28 @@ def cli(args: list[str]):
         "quickstart", help='Create a "project.skore" file and start the UI'
     )
 
+    if args is None:
+        args = sys.argv[1:]
```

Comment:
I prefer the previous version, where the caller was in charge of passing parameters.

Reply:
When creating the entry point, the executable that you get is the following:

```shell
#!/Users/glemaitre/Documents/packages/src/skore/.pixi/envs/dev/bin/python3.12                                                                                                                
# -*- coding: utf-8 -*-                                                                                                                                                                      
import re
import sys
from skore.__main__ import cli
if __name__ == "__main__":
    sys.argv[0] = re.sub(r"(-script\.pyw|\.exe)?$", "", sys.argv[0])
    sys.exit(cli())
```

So it means that no `args` are parsed explicitly. Are you aware of a way to pass the args explicitly in the entry point?

Reply:
It's just a minor change but i would do something like (and take advantage of `rich` even in entry-point):

```python
# skore/src/skore/__main__.py

import sys
from skore.cli.cli import cli

def main():
    import rich.traceback
    
    # Display error tracebacks with Rich
    rich.traceback.install(show_locals=True)

    sys.exit(cli(sys.argv[1:]))
    
if __name__ == "__main__":
    main()
```

```toml
# skore/pyproject.toml

[project.scripts]
skore = "skore.__main__:main"
```

```python
# skore/src/skore/cli/cli.py

def cli(args: list[str]):
[...]
```